hackathon_site/event/jinja2/event/change_password.html

```diff
@@ -0,0 +1,33 @@
+{% extends "event/form_base.html" %}
+
+{% block form_title %}Change Password{% endblock %}
+
+
+{% set form_action=url("event:change_password") %}
+               
+{% block form_inputs %}
+    {{ render_field(form.old_password) }}
+    {{ render_field(form.new_password1, show_help_text=True) }}
+    {{ render_field(form.new_password2) }}
+
+    {% if form.non_field_errors %}
+        {% for error in form.non_field_errors() %}
+            {{ error }}
+        {% endfor %}
+    {% endif %}
+{% endblock %}
+
+{% block form_button %}
+<button type="submit" class="btn-small waves-effect waves-light secondaryColor colorBtn center">Change Password</button>
+{% endblock %}
+
+{% block form_bottom %}
+<div class="col s12 loginLink">
+    <i class="material-icons primaryText">exit_to_app</i>
+    <a href="{{ url("event:dashboard") }}" class="primaryText hoverLink s6">Back to Dashboard</a>
+</div>
+<br />
+<div class="row center">
+    Something wrong? <a href="#" class="primaryText hoverLink">Contact us</a>.
```

Comment:
Add the `CONTACT_EMAIL` from settings into here, like some of the other templates do. You can add it in through the `extra_context` parameter to `.as_view()` (see registration/urls.py for an example), though since we need it in a lot of templates I'd recommend adding `from_email` and `contact_email` to our default Jinja2 environment (hackathon_site/jinja2.py), so they're available everywhere.